### PR TITLE
Migrate PostgreSQL modules

### PR DESCRIPTION
**Pull request by kvch**
In the list of Integrations:

image

Overview of the integration:

image

Part of fields and example event in description:

image

Configuration of logging:

image

Configuration of metrics:

image

Postgresql logs overview:

image

Postgresql metrics overview:

image

### CODE REVIEW

**Comment by mtojek** [comment]
@kvch

Thank for your contribution. Is this still a draft or it's ready to be reviewed? Feel free to add label for the team.

It would great to see some screenshots from Kibana (PR description) if this integration is running. We used to take screenshots presenting configuration options and working dashboards.

**Comment by mtojek** [comment]
Based on the posted screenshots of forms, it's in the really good shape! Did you try with dashboards, @kvch ?

**Comment by kvch** [comment]
@mtojek I've just added all required screenshots. This PR is ready for review.

File: packages/postgresql/dataset/activity/agent/stream/stream.yml.hbs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+metricsets: ["activity"]
+hosts:
+{{#each hosts}}
+  - {{this}}
+{{/each}}
+username: {{username}}
```
[the remark this answers is not in the document]
Comment (kvch) [reply packages/postgresql/dataset/activity/agent/stream/stream.yml.hbs:6]: What do you mean? If I don't add these, metricbeat is not picking up these settings.

[the remark this answers is not in the document]
Comment (mtojek) [reply packages/postgresql/dataset/activity/agent/stream/stream.yml.hbs:6]: It depends if they're mandatory or not (whether Metricbeat starts without them).

By custom I mean wrapping them with the `if` condition: `{{#if variablem}}`

**Comment by mtojek** [comment]
I'm not quite sure about the SIGSEGV issue on the CI side. Could you please rerun tests?

```
[2020-06-25T17:33:55.863Z] [signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x5cfe05]

[2020-06-25T17:33:55.863Z] 

[2020-06-25T17:33:55.863Z] goroutine 1 [running]:

[2020-06-25T17:33:55.863Z] main.init.0()

[2020-06-25T17:33:55.863Z] 	/go/src/golang.org/x/tools/cmd/goimports/goimports.go:51 +0x85
```

**Comment by kvch** [comment]
@mtojek I've addressed your remaining concerns. Also, the CI issue seems transient. It is no longer a problem.

**Pull request merged**